src/calva-fmt/src/infer.ts

```diff
@@ -147,4 +147,6 @@ function applyResults(r: ResultOptions, editor: vscode.TextEditor) {
     }
 }
 
-export function updateState(editor: vscode.TextEditor) {}
+export function updateState(editor: vscode.TextEditor) {
+    // do nothing
+}
```

Comment:
It's better to have some sort of comment making it clear that doing nothing is intentional.

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-empty-function.md